sql/src/test/java/io/crate/analyze/CopyAnalyzerTest.java

```diff
@@ -154,8 +155,8 @@ public void testCopySysTableTo() throws Exception {
     public void testCopyToWithColumnList() throws Exception {
         CopyAnalyzedStatement analysis = (CopyAnalyzedStatement)analyze("copy users (id, name) to DIRECTORY '/tmp'");
         assertThat(analysis.selectedColumns().size(), is(2));
-        assertThat(((Reference)analysis.selectedColumns().get(0)).info().ident().columnIdent().name(), is("id"));
-        assertThat(((Reference)analysis.selectedColumns().get(1)).info().ident().columnIdent().name(), is("name"));
+        assertThat(analysis.selectedColumns().get(0), isReference("id"));
+        assertThat(analysis.selectedColumns().get(1), isReference("name"));
```

Comment:
will keep isReference, because with isSQL the test would also pass if the columns are Fields